composer.json

```diff
@@ -17,18 +17,18 @@
     "require": {
         "php": ">5.4",
         "paragonie/random_compat": "~1.0|~2.0|~9.99.99",
-        "symfony/framework-bundle": "~2.3|~v3.0|~4.0|~5.0",
-        "symfony/security-core": "~2.3|~3.0|~4.0|~5.0",
-        "symfony/security-http": "~2.3|~3.0|~4.0|~5.0",
-        "symfony/security-csrf": "~2.3|~3.0|~4.0|~5.0",
-        "symfony/yaml": "~2.3|~3.0|~4.0|~5.0",
+        "symfony/framework-bundle": "~2.3|~v3.0|~4.0|~5.0|~6.0",
```

Comment:
We can safely use `"^4.4 || ^5.3 || ^6.0"` here. Especially because this bundle is about security, we should drop Symfony versions that are EOL and won't get security fixes. See https://symfony.com/releases